SEED-platform/seed
# Update the BPS Sample Data AT Files

#### Any background context you want to provide?
The property name in the AT files hadn't been updated yet. Additionally, there was meter data associated with one of the files that was still from the source AT property. 

#### What's this PR do?
I updated the property names and removed the meter data.

Additionally, I added a disclaimer to the readme indicating that only the identifiable data in the AT files were revised to match the sample BPS data.

#### How should this be manually tested?
Try importing an AT file and make sure that the property name matches the BPS sample data.

#### What are the relevant tickets?
Fixes #3478

#### Screenshots (if appropriate)
